src/main/java/com/algorand/algosdk/util/Uint64Encoder.java

```diff
@@ -0,0 +1,67 @@
+package com.algorand.algosdk.util;
+
+import java.math.BigInteger;
+
+public class Uint64Encoder {
```

Comment:
nit: I'd probably put this in `Encoder` so that it shows up in IDEs with `Encoder.<tab>`